jekyll/_sass/_sidebar.scss

```diff
@@ -142,46 +141,18 @@ $sidebar-spacing-base: 5px;
 }
 
 .sidebar-mobile-wrapper {
-  border-top: 1px solid $gray-200;
-  border-bottom: 1px solid $gray-200;
-  padding: 50px 0 0 0;
-  width: 100%;
-  background: $gray-100;
   display: none;
-  cursor: pointer;
-
-  .icon {
-    width: 24px;
-  }
-
-  .mobile-sidebar-item {
-    display: block;
-    width: 100%;
-    padding: $spacing-small-vertical 16px;
-
-    &.hidden {
-      display: none;
-    }
-
-    span {
-      display: inline-block;
-      margin-left: $spacing-small-horizontal;
-    }
-
-    .arrow {
-      float: right;
-      margin-top: 5px;
-      transform: rotate(90deg);
-    }
-  }
+  width: 100%;
 
   .mobile-sidebar {
     @extend .sidebar;
-    padding-right: 16px;
```

Comment:
This padding-right I think was preventing the open arrows from overflowing, and creating a horizontal scrollbar.

![Screen Shot 2021-04-26 at 11 43 32 AM](https://user-images.githubusercontent.com/5113432/116113655-ae98a580-a686-11eb-8788-f17661c8a76e.png)

![Screen Shot 2021-04-26 at 11 43 42 AM](https://user-images.githubusercontent.com/5113432/116113640-ab051e80-a686-11eb-905b-c93a86a774b6.png)